pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -1041,7 +1041,7 @@ self: {
        libraryHaskellDepends = [
          base bytestring mtl process template-haskell
        ];
-       description = "A simple music library with the capability of generating .ly and .mid files.";
```

Comment:
This file is automatically generated, so these changes would just get overwritten next time it is generated.

These descriptions are taken directly from the upstream packages.  If you wanted to fix this, you'd have to submit a PR to https://github.com/NixOS/cabal2nix (which is the tool that does the generations).